website/src/pages/index.js

```diff
@@ -36,7 +35,7 @@ class Index extends React.Component {
 
 		return (
 			<section>
```

Comment:
you need to apply the theme/Layout component around all your pages, otherwise they won't have any layout